generator/src/lib.rs

```diff
@@ -1798,6 +1798,25 @@ fn derive_default(
     Some(q)
 }
 
+fn derive_send_sync(struct_: &vkxml::Struct, has_lifetime: bool) -> Option<TokenStream> {
+    if !struct_
+        .elements
+        .iter()
+        .filter_map(get_variant!(vkxml::StructElement::Member))
+        .any(|e| e.reference.is_some())
+    {
+        // No pointers, no need to implement Send/Sync
+        return None;
+    }
+    let name = name_to_tokens(&struct_.name);
+    let lifetime = has_lifetime.then(|| quote!(<'_>));
```

Comment:
Won't `has_lifetime` always be true at this point? We know this struct has pointers, so why bother checking that flag?

Reply:
Good point. That was non-obvious to me. Updated.